### PR TITLE
Compute SpecAug in jnp.bool (1 byte), not jnp.int32.

### PR DESCRIPTION
**Pull request by ds-hwang**
It saves the temp memory for a [batch, time, freq] mask.

### CODE REVIEW

**Comment by ds-hwang** [comment]
@markblee Could you review it? From 1351

**Review by markblee — APPROVED** [review]
